servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/CheckContent.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.quarkus.cli;
+
+import com.google.protobuf.ByteString;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+import org.projectnessie.versioned.GetNamedRefsParams;
+import org.projectnessie.versioned.Hash;
+import org.projectnessie.versioned.Key;
+import org.projectnessie.versioned.ReferenceInfo;
+import org.projectnessie.versioned.ReferenceNotFoundException;
+import org.projectnessie.versioned.StoreWorker;
+import org.projectnessie.versioned.persist.adapter.ContentAndState;
+import org.projectnessie.versioned.persist.adapter.KeyFilterPredicate;
+import org.projectnessie.versioned.persist.adapter.KeyListEntry;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "check-content",
+    mixinStandardHelpOptions = true,
+    description = "Check content readability of active keys.")
+public class CheckContent extends BaseCommand {
+
+  @CommandLine.Option(
+      names = {"-s", "--separator"},
+      defaultValue = ": ",
+      description = "Output field separator.")
+  private String separator;
+
+  @CommandLine.Option(
+      names = {"-k", "--key"},
+      description =
+          "Content key element (zero or more). If not set, all current keys will be checked.")
```

Comment:
Nit: `elements`?

Reply:
Hm - can we use `UriUtil` here to make it a bit more user friedly?

Reply:
I'll add a separate option for the encoded key form. In my experience using multiple options (one per key element) is sometimes very handy, e.g. when the user already has a key that is complex and not encoded.

Reply:
On the second thought, I guess using encoded keys may not be very convenient in this situation. Most (if not all) other Nessie CLI tools produce unencoded keys (including this command that now produce JSON arrays for keys). So for an end user I think it would be more convenient to copy-paste that data into individual `-k` options. WDYT?

Reply:
WFM